state/execution.go

```diff
@@ -303,34 +303,50 @@ func execBlockOnProxyApp(
 	if pbh == nil {
 		return nil, errors.New("nil header")
 	}
+<<<<<<< HEAD
```

Comment:
?????

Reply:
The mergify action does this on backports where there are merge conflicts. I wish it didn't commit the conflicts (/cc @marbar3778). I'll fix these soon.

Reply:
I dont think there's a way to avoid this, the bot doesn't know which is right. Unless AI... 🤔